code/modules/mob/living/carbon/human/life/handle_regular_status_updates.dm

```diff
@@ -158,11 +151,16 @@
 		protection_aura = protection_new
 		marskman_aura = marskman_new
 
-		//hud_set_pheromone() //TODO: HOOK THIS UP, ASK PHIL
-
 		mobility_new = 0
 		protection_new = 0
 		marskman_new = 0
+		
+		//Natural recovery; enhanced by hold/protection aura.
+		var/aura_recovery_multiplier = 1 + max(0,0.5 * protection_aura) //Protection aura adds +50% recovery rate per point of leadership beyond the first; +150% for a CO/XO
```

Comment:
Make this recovery mirror the frenzy bonus to cancel it out.

Reply:
Frenzy bonus is a bonus to the damage.

Protection aura provides a bonus to the recovery rate.

These things can't really be the same, unless you want the protection aura to reduce the incoming tackle damage by the same amount as the Frenzy.